app/src/main/java/com/babylon/wallet/android/presentation/transaction/analysis/GeneralTransferAnalysis.kt

```diff
@@ -73,9 +74,11 @@ suspend fun TransactionType.GeneralTransaction.resolve(
         isRefreshing = false
     ).value().orEmpty()
 
+    val defaultDepositGuarantee = getProfileUseCase.defaultDepositGuarantee().toFloat()
```

Comment:
Im not entirely sure if that is the best place to provide this input down to the model but let me know otherwise